src/locales/de/trainers.ts

```diff
@@ -19,6 +19,12 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "Galaktik-Boss",
   "plasma_boss": "Weiser von Team Plasma", // This is on purpose, since "Ghetsis" is never mentioned as the boss of team plasma in the game but as "Weiser"
   "flare_boss": "Flare-Boss",
+  "aether_boss": "Æther-Präsidentin",
+  "aether_admin": "Aether Foundation Admin",
+  "skull_boss": "Skull-Boss",
+  "skull_admin": "Team Skull Admin",
+  "macro_boss": "Geschäftsführer von Macro Cosmos",
+  "macro_admin": "Macro Cosmos"
```

Comment:
```suggestion
  "aether_boss": "Æther-Präsidentin",
  "aether_admin": "Æther-Regionalleiter",
  "skull_boss": "Skull-Boss",
  "skull_admin": "Team Skull Vorstand",
  "macro_boss": "Geschäftsführer von Macro Cosmos",
  "macro_admin": "Vizepräsidentin von Macro Cosmos"
```

Note: The Title for Faba & Oleana are for their respective characters. They probably would not fit if we ever add other admins.